test/pseudo-tty/ref_and_unref.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const TTY = process.binding('tty_wrap').TTY;
+
+const handle = new TTY(0);
+handle.readStart();
+
+handle.unref();
+handle.ref();
+
+var timeout = setTimeout(function() {
```

Comment:
@cjihrig Hmmm, I like the timer check here tbh.

I think `_getActiveHandles()` will be hard to filter though since there will also be process.stdout at some point, and maybe a timer handle if we don't use `_unrefActive()`

(I've never had much checking `_getActiveHandles()` in a useful way I guess?)